src/Ieemdb.Web/Fakers.cs

```diff
@@ -35,6 +35,10 @@ public static class Fakers
         .RuleFor(e => e.Bio, f => f.Lorem.Sentence(5, 20))
         .RuleFor(e => e.Id, f => f.IndexFaker);
 
+    public static readonly Faker<CountryDto> CountryProvider =
+      new Faker<CountryDto>()
+        .RuleFor(e => e.CountryOrigin, f => f.Person.FirstName);
```

Comment:
.RuleFor(e => e.CountryOrigin, f => f.Address.Country);
Doesnt work, so instead a name can work just fine

Reply:
Who cares it's just a faker.